tsconfig.json

```diff
@@ -1,10 +1,13 @@
 {
     "compilerOptions": {
-        "target": "es5",
-        "module": "commonjs",
-        "esModuleInterop": true,
+        "target": "ES2022",
+        "module": "ES2020",
+        "outDir": "./dist/esm",
         "declaration": true,
-        "outDir": "./dist",
+        "declarationDir": "./dist",
```

Comment:
unless this enables them another way

Reply:
This just sets the declaration directory to `dist` as it was, as `tsc` now also emits the ESM files into `dist/esm` the declarations have to explicitly emitted into `./dist`